dask_jobqueue/pbs.py

```diff
@@ -60,66 +62,44 @@ class PBSCluster(JobQueueCluster):
 
     >>> cluster.adapt()
     """
+
+    _submitcmd = 'qsub'
+    _cancelcmd = 'qdel'
+
+
     def __init__(self,
-                 name='dask',
-                 queue='regular',
+                 name='dask-worker',
+                 queue=None,
                  project=None,
-                 resource_spec='select=1:ncpus=36:mem=109GB',
-                 threads_per_worker=4,
-                 processes=9,
-                 memory='7GB',
+                 resource_spec='select=1:ncpus=24:mem=100GB',
                  walltime='00:30:00',
-                 interface=None,
-                 death_timeout=60,
-                 extra='',
+                 job_extra=[],
+                 local_directory='$TMPDIR',
                  **kwargs):
-        self._template = """
-#!/bin/bash
-
-#PBS -N %(name)s
-#PBS -q %(queue)s
-#PBS -A %(project)s
-#PBS -l %(resource_spec)s
-#PBS -l walltime=%(walltime)s
-#PBS -j oe
-
-%(base_path)s/dask-worker %(scheduler)s \
-    --nthreads %(threads_per_worker)d \
-    --nprocs %(processes)s \
-    --memory-limit %(memory)s \
-    --name %(name)s-%(n)d \
-    --death-timeout %(death_timeout)s \
-     %(extra)s
-""".lstrip()
-
-        if interface:
-            host = get_ip_interface(interface)
-            extra += ' --interface  %s ' % interface
-        else:
-            host = socket.gethostname()
+
+        super(PBSCluster, self).__init__(name=name, local_directory=local_directory, **kwargs)
 
         project = project or os.environ.get('PBS_ACCOUNT')
-        if not project:
-            raise ValueError("Must specify a project like `project='UCLB1234' "
-                             "or set PBS_ACCOUNT environment variable")
-        self.cluster = LocalCluster(n_workers=0, ip=host, **kwargs)
-        memory = memory.replace(' ', '')
-        self.config = {'name': name,
-                       'queue': queue,
-                       'project': project,
-                       'threads_per_worker': threads_per_worker,
-                       'processes': processes,
-                       'walltime': walltime,
-                       'scheduler': self.scheduler.address,
-                       'resource_spec': resource_spec,
-                       'base_path': dirname,
-                       'memory': memory,
-                       'death_timeout': death_timeout,
-                       'extra': extra}
-        self.jobs = dict()
-        self.n = 0
-        self._adaptive = None
-        self._submitcmd = 'qsub'
-        self._cancelcmd = 'qdel'
+
+        header_lines = ['#PBS -N %s' % name]
+        if queue != None: header_lines.append('#PBS -q %s' % queue)
+        if project != None: header_lines.append('#PBS -A %s' % project)
+        if resource_spec != None: header_lines.append('#PBS -l %s' % resource_spec)
+        if walltime != None: header_lines.append('#PBS -l walltime=%s' % walltime)
```

Comment:
Most other dask related projects use the `flake8` style checker, which would probably raise some warnings here for combining the if block into a single line.  You might want to run the following from the root of this directory:

    flake8 dask_jobqueue

But we should probably add this to CI so that we can automate this.

Reply:
That's fine with me. Perhaps we could enable Travis CI to do some basic tests and style checking, without having all the docker testing env yet? Should we raise an issue separated from #2 ?

Reply:
I've done this in #13. You can also run the same checker locally by running: `flake8 -j auto dask_jobqueue`